canonical/maas-ui
# Display bios boot mode in node overview card

## Note

The commits in this PR are being merged in #4377.

## Done

- Added "BIOS boot mode" to the node hardware information card
- Removed "Domain" from the node overview card, since it's displayed in the header next to the node name
- Migrated DetailsCard.test to RTL 
- Migrated HardwareCard.test to RTL and removed need for snapshots
- Migrated NumaCard.test to RTL and removed need for snapshots
- Migrated NumaCardDetails.test to RTL and removed need for snapshots
- Added lines in between list items on the Numa Node and Hardware Information cards
- Removed lines between card titles and content
- Shrunk top padding on all cards in Machine Summary
- Left-aligned list item labels in Numa Node and Hardware Information cards

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

1. Log in and navigate to Machines
2. Click a machine and ensure that the BIOS boot mode displays in the hardware info card. _Note: this may display as `Unknown` if no mode is set_
3. Navigate to Controllers
4. Click a controller and ensure that the BIOS boot mode displays in the hardware info card. _Note: this may display as `Unknown` if no mode is set_

## Fixes

Fixes #1595 

## Launchpad issue

`lp#1893670`
https://bugs.launchpad.net/maas/+bug/1893670

## Screenshots

### Before

![image](https://user-images.githubusercontent.com/35104482/186399194-83d38695-2fa2-4f94-afdb-7b2e2a722076.png)

### After

![image](https://user-images.githubusercontent.com/35104482/186399282-60bcd98b-7033-4f30-90bb-882f0af5123c.png)
